### PR TITLE
Dd 210 reference content documentation

### PR DESCRIPTION
**Pull request by rafal-sereda-droptica**

### CODE REVIEW

File: themes/custom/droopler_theme/scss/layout/_modifiers.scss [hunk 1]
```diff
@@ -375,6 +375,6 @@ $two-columned-header-line-margin: 0.625rem !default;
 
 section.header-margin-none {
   .field--name-field-d-main-title {
-    margin-bottom: inherit;
+    margin-bottom: inherit !important;
```
Comment (rafal-sereda-droptica) [inline themes/custom/droopler_theme/scss/layout/_modifiers.scss]: :pray: 
i know!
just :see_no_evil:  
:pray:

Comment (dominika-rozycka-droptica) [reply themes/custom/droopler_theme/scss/layout/_modifiers.scss]: :(

Comment (rafal-sereda-droptica) [reply themes/custom/droopler_theme/scss/layout/_modifiers.scss]: fixed, no sad face today!

File: themes/custom/droopler_theme/scss/components/_d_p_text_blocks.scss [hunk 2]
```diff
@@ -62,6 +62,10 @@ $d-p-single-text-box-grid-large-padding: $grid-gutter-width !default;
       @include media-breakpoint-down(md) {
         margin: $d-p-text-blocks-mobile-group-margin;
       }
+
+      @at-root section.header-margin-none & {
```
Comment (dominika-rozycka-droptica) [inline themes/custom/droopler_theme/scss/components/_d_p_text_blocks.scss]: :heart:

**Pull request merged**